sound/soc/sof/ipc4.c

```diff
@@ -637,6 +653,7 @@ static void sof_ipc4_rx_msg(struct snd_sof_dev *sdev)
 		sof_ipc4_mtrace_update_pos(sdev, SOF_IPC4_LOG_CORE_GET(ipc4_msg->primary));
 		break;
 	case SOF_IPC4_NOTIFY_EXCEPTION_CAUGHT:
+		sof_ipc4_create_exception_debugfs_node(sdev);
```

Comment:
what will happen if you catch another exception by powering off and then booting the DSP again?

Reply:
Good catch! Update it that only create one entry for exception.  First I created the debug node in fw_ready, but then I found that telemetry slot is only allocated when FW panic happen. That is why I first used fixed slot id.

Reply:
Hrm, right, we are re-purposing the TELEMETRY slot for exception dump.